src/System.Reflection/tests/GetTypeTests.cs

```diff
@@ -12,6 +12,14 @@ public class GetTypeTests
         [Fact]
         public void GetType_EmptyString()
         {
+#if MONO
+            string exceptionParamName = "typeName";
+            string exceptionParamName2 = "name";
```

Comment:
We should unify them as well to be `typeName`

Reply:
done